source/hardware/controllers/yaeltex_minimixxx.rst

```diff
@@ -0,0 +1,312 @@
+Or, press the currently-activated layer button to disable it.
+
+In the Default Layer, the encoders above the Pitch sliders will have an animated
+spinning indicator, and the encoders above the Level sliders will show an
+animated VU meter for each deck.
+
+===============================  ========  =====================================
+Button                           Category  Function
+===============================  ========  =====================================
+:hwlabel:`2`                     Deck 1    :hwlabel:`KEYLOCK`. Tapping this button toggles keylock. Press and hold this button and move the :hwlabel:`PITCH` slider to adjust musical key without changing track speed.
+:hwlabel:`3`                     Deck 1    Toggles :hwlabel:`FX` Unit 1 for each deck.
+:hwlabel:`10`                    Deck 1    :hwlabel:`SYNC`.  Tap to perform a one-off beatsync. Press and hold to enable Sync Lock.
+:hwlabel:`11`                    Deck 1    Enables the :hwlabel:`LOOP` layer for each individual deck.
+..
+:hwlabel:`8`                     Deck 2    Enables the :hwlabel:`HOTCUE` layer for each deck.
+:hwlabel:`5`                     Deck 2    :hwlabel:`KEYLOCK`. Tapping this button toggles keylock. Press and hold this button and move the :hwlabel:`PITCH` slider to adjust musical key without changing track speed.
+:hwlabel:`6`                     Deck 2    Toggles :hwlabel:`FX` Unit 1 for each deck.
+:hwlabel:`13`                    Deck 2    :hwlabel:`SYNC`.  Tap to perform a one-off beatsync. Press and hold to enable Sync Lock.
+..
+:hwlabel:`7`                     Deck 1    Enables the :hwlabel:`HOTCUE` layer for Deck 1.
+:hwlabel:`8`                     Deck 2    Enables the :hwlabel:`HOTCUE` layer for Deck 2.
+:hwlabel:`15`                    Layer     Enables the :hwlabel:`LIBRARY` layer. Hold :hwlabel:`SHIFT` and press to enable the :hwlabel:`MAIN GAIN` layer.
+:hwlabel:`16`                    Layer     :hwlabel:`SHIFT`
+===============================  ========  =====================================
+
+==================  ================  ========================  ========================================
+Encoder             Category          Action                    Function
+==================  ================  ========================  ========================================
+Encoder 1           Deck 1 Jog        Spin                      Jog forward and back, or nudge faster or slower if the track is playing.
+..                  ..                :hwlabel:`SHIFT` + Spin   Seek forward a large distance in the track.
+..                  ..                Press                     Creates a loop if not in a loop, or reloop toggle if the playhead is in the loop.
+..                  ..                :hwlabel:`SHIFT` + Press  Does a beatloop roll.
+Encoder 2           Deck 1 Pregain    Spin                      Adjust track pregain.
+..                  ..                Press                     Toggle pfl.
```

Comment:
```suggestion
..                  ..                Press                     Toggle :term:`pfl`.
```
Will result in a link to the glossary explaining what pfl is.

Reply:
done (manually)